### PR TITLE
[12.0][MIG] base_sparse_field_list_support

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: base_sparse_field_list_support/__manifest__.py [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-    "version": "10.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Technical Settings",
     "website": "www.akretion.com",
     "author": "Akretion",
     "license": "AGPL-3",
     "application": False,
-    "installable": False,
+    "installable": True,
     "depends": ["base"],
```
Comment (lmignon) [inline base_sparse_field_list_support/__manifest__.py]: @rousseldenis you must depends on [base_sparse_field](https://github.com/odoo/odoo/tree/12.0/addons/base_sparse_field)

**Review by lmignon — CHANGES_REQUESTED** [review]
@rousseldenis Travis is :apple:  I think you must modify travis to use  postgresql: "9.6"
```yaml

addons:
  postgresql: "9.6"
  apt:
    packages:
      - expect-dev  # provides unbuffer utility
```

**Comment by rousseldenis** [comment]
> @rousseldenis Travis is  I think you must modify travis to use postgresql: "9.6"
> 
> ```yaml
> addons:
>   postgresql: "9.6"
>   apt:
>     packages:
>       - expect-dev  # provides unbuffer utility
> ```

That's what I was looking for! Thanks :smiley:

File: base_sparse_field_list_support/__manifest__.py (hunk 1 above)
Comment (rousseldenis) [reply base_sparse_field_list_support/__manifest__.py]: Ok, integrated in Odoo now.

**Comment by rousseldenis** [comment]
@lmignon Done and green

**Review by lmignon — APPROVED** [review]

**Pull request merged**